docs/user/getting-started.md

```
@@ -199,52 +226,43 @@ Finally, move it into a location within your shell's search path.
-[...]
-```
-
-In the web browser that opens, use your ACM login credentials.
+You will be logging in with ACM login credentials.
 
-Verify you can now access the service via the CLI:
+Login using a user token:
 
 ```console
-$ flightctl get devices
+$ flightctl login ${FC_API} -t $(oc whoami -t)
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)